connect-inject/endpoints_controller.go

```diff
@@ -413,7 +415,7 @@ func (r *EndpointsController) processUpstreams(pod corev1.Pod) ([]api.Upstream,
 // getConsulClient returns an *api.Client that points at the consul agent local to the pod.
 func (r *EndpointsController) getConsulClient(ip string) (*api.Client, error) {
```

Comment:
I liked Luke's suggestion from [here](https://github.com/hashicorp/consul-k8s/pull/470#pullrequestreview-628110014) to name this `remoteConsulClient` or `externalConsulClient`

Reply:
`remoteConsulClient` looks great, switching to that, thanks!